### ISSUE TITLE
Logo is double, editing component stops unwanted

### ISSUE THREAD

**Issue by LWA004** [chuanqisun/figma-plugin-spacemouse#18]
Because I use SketchUp occasionally, I only use the free web-version.
Unfortunately I bought a 3D Connection Mouse, not being aware that it only works within the pro-version of SketchUp (big disappointment).
Luckely your solution helped. Thank you for that.

But there are 2 issues I discovered the first day of use.

1. I see the old SketchUp logo overlapping the current one from Trimble at the right top of my screen. The one from Trimble cannot be turned off (only in the payed version I assume). Hopefully the old logo is placed by your driver and can be deleted?

2. When editting a component and turning it around by the 3D mouse, sometimes/often the editing stops unexpectedly, while I didn't wanted it to. Is this a known bug? Or am I doing something wrong?

Hope to hear from you.

**Comment by chuanqisun** [comment]
Could you share a screenshot of the logo issue for reference? Thank you!

**Comment by LWA004** [comment]
Hello Chuánqí Sun,

Sorry for my late reply, but I had to give priority to other things.
And sorry again, because I'm not able to reproduce the double logo anymore.
So probably it only was when I first installed your free software it was an
issue. Today it's working fine.
Whenever it pops up again, I'll let you know.

Yours Sincerely,
Leo van der Wal,
Holland.

Op vr 19 jul 2024 om 18:45 schreef Chuánqí Sun ***@***.***>:
